apps/web/app/(app)/environments/[environmentId]/settings/api-keys/components/EditApiKeys.tsx

```diff
@@ -59,6 +60,26 @@ export default function EditAPIKeys({
     }
   };
 
+  const ApiKeyDisplay = ({ apiKey }) => {
+    const copyToClipboard = () => {
+      navigator.clipboard.writeText(apiKey);
+      toast.success("API Key copied to clipboard");
+    };
+
+    if (!apiKey) {
+      return <span className="italic">secret</span>;
+    }
+
+    const displayKey = `${apiKey.slice(0, 3)}...${apiKey.slice(-3)}`;
```

Comment:
I would prefer if we don't truncate the text to still have a fallback that a user can copy the key manually or recheck if the copied & pasted key is correct. Can you also make it work without the truncate? 😊